docs/resource-collection.rst

```diff
@@ -22,7 +22,8 @@ The nextstrain.org testing & production configs currently set this to
 ``s3://nextstrain-inventories/resources/v<revision_number>.json.gz``.
 
 If you make any updates that changes the structure or the contents of the resource
-index JSON, then bump the ``<revision_number>`` within the configs (``env/production/config.json``)
+index JSON, then bump the ``<revision_number>`` within the configs
+(``env/testing/config.json`` and ``env/production/config.json``)
```

Comment:
Is there a situation where the `<revision_number>` should differ in these files?

Reply:
Hmm, `env/testing/config.json` is only used for local testing right now so I don't think so? 
(Maybe if we ever set up a completely separate testing instance on AWS there would be reason for them to differ?)

Reply:
Ok - in this case [I continue to think](https://github.com/nextstrain/nextstrain.org/issues/980#issuecomment-2282869865) that we should add a check in CI that they are the same. Otherwise it's too easy to overlook the docs, get things working locally, and then overwrite the production version.

Reply:
Good suggestion! I added in https://github.com/nextstrain/nextstrain.org/pull/983/commits/9d45dd059d4712d997c9a57229294999ae023565